### ISSUE TITLE
Controls are not displayed

### ISSUE THREAD

**Issue by nielsgeo** [elb-min-uhh/elearnvideo.js#1]
Since recently, the controls of video, for example play-button, have not been displayed anymore. There are only boxes, see attached picture or [here.](https://rlab.blogs.uni-hamburg.de/dig-skripte/Vorstellung_Daten_Kaule/index.html?p=3) 

I tested this in chrome (PC, windows 7) and firefox (mobile, android).

Is this an issue or did we do anything wrong with updates or code?

 ![video](https://user-images.githubusercontent.com/22545097/48853236-5707f880-edaf-11e8-881c-e6fed9d17906.PNG) #

**Comment by westarne** [comment]
The console output of the linked page says, that the `eLearn-Video` font files are missing.

* `assets/font/eLearn-Video.ttf`
* `assets/font/eLearn-Video.woff`

Using [atom-elearnjs](https://github.com/elb-min-uhh/atom-elearnjs) v0.6.3 and [vsc-elearnjs](https://github.com/elb-min-uhh/vsc-elearnjs) v1.0.12 I could not reproduce an export missing these files. If you're using one of these plugins please try the export again activating `Export elearn.js assets` and `Include elearnvideo.js` and check if the font files can be found in the described paths within the export, to check if there's a problem with the package.

If this exports correctly you can simply use this version.
Please let me know if you can reproduce an incorrect export with this missing files.

If you do not use the plugins, you should simply be able to fix this by copying the files from the repository to your directory. But I would recommend checking the packages as they let you focus on your content without having to worry about the file export and up to date versions.

---

Also you seem to be using an older modified version of the `template.html` from the elearn.js repository. This includes the quiz.js in the `<head>` even though you do not provide (and need) this file. So it is not found like the font files.

If you want to switch to a Markdown file converting to the final file (with one of our plugins), you can use the `meta` and `imprint` described here: https://github.com/elb-min-uhh/markdown-elearnjs/wiki/Syntax-Extensions

In your case a basic meta block for example would be:

```
<!--meta
    Title: Vorstellung der ,,soil&rdquo;- und ,,veg&rdquo;-Datensätze
    Author: Niels Schwab | Melanie Werner
    Description: Vorstellung von Datensätzen des RLab
    Keywords: "datensatz, vorstellung, einführung, bodenkundliche daten, vegetationskundliche daten, kaule, nepal, deutsch, german, RStudio, R, geographie, Statistik, R-Statistik, R statistics, R Project, R, R selbst lernen, R selbstlernen, R selbstgesteuert lernen, R blended learning, R flipped classroom, RStudio selbst lernen, RStudio selbstlernen, RStudio selbstgesteuert lernen, RStudio blended learning, RStudio flipped classroom"
    Custom: '<script>
                $(document).ready(function() {
                    eLearnJS.setBackButtonEnabled(true);
                    eLearnJS.setBackButtonText("RLab Home");
                    eLearnJS.setBackPage("http://rlab.blogs.uni-hamburg.de", "link");
                });
            </script>'
-->
```

**Comment by nielsgeo** [comment]
Thanks for the swift reply, support and extensive additional information! The problem was solved by manually adding the missing font files. 

Since the Digital Skript is an old one and we do not have markdown file we did not use the new "atom way" to produce it. There was only one minor change done and we thought it is good idea to update to the current versions of elearn.js and elearn-video at the same time. Somehow the fonts were missed...

To produce new Digital Skripts we use the atom plugin and it is great improvement compared to the procedure. At that time I found it easier to write directly in html, based on the template and copy&paste from other scripts, without using markdown.
